src/engine/source/indexerconnector/src/indexerConnector.cpp

```diff
@@ -106,7 +108,10 @@ static void builderBulkIndex(std::string& bulkData, std::string_view id, std::st
 IndexerConnector::IndexerConnector(const nlohmann::json& config, const uint32_t& timeout, const uint8_t workingThreads)
 {
     // Get index name.
+    auto currentTime = base::utils::time::getCurrentTimestamp(true);
+    Utils::replaceAll(currentTime, "/", ".");
     m_indexName = config.at("name").get_ref<const std::string&>();
+    Utils::replaceAll(m_indexName, "$(date)", currentTime);
```

Comment:
And with the proposal above
```suggestion
    Utils::replaceAll(m_indexName, "$(date)", base::utils::time::getCurrentDate());
```

Reply:
Done